### PR TITLE
Added military address in CC

### PR DESCRIPTION
**Pull request by adeelehsan**
military state options added in cyber Source

### CODE REVIEW

File: ecommerce/extensions/payment/forms.py [hunk 1]
```diff
@@ -167,16 +169,16 @@ def clean(self):
...
-                    # is fixed.
-                    if not pycountry.subdivisions.get(code=code):
-                        raise KeyError
-                except KeyError:
-                    msg = _('{state} is not a valid state/province in {country}.').format(state=state, country=country)
-                    logger.debug(msg)
-                    raise ValidationError({'state': msg})
+                if country != 'US' and state not in US_MILITARY_ADDRESS:
+                    try:
+                        # TODO: Remove the if statement once https://bitbucket.org/flyingcircus/pycountry/issues/13394/
```
Comment (waheedahmed) [inline ecommerce/extensions/payment/forms.py:175]: This issue has been resolved, after verifying can you please also remove the following if statement?

**Review by waheedahmed — COMMENTED** [review]
Add a test case otherwise LGTM 👍

File: ecommerce/extensions/payment/tests/test_forms.py [hunk 2]
```diff
@@ -183,6 +183,9 @@ def test_organization_field_in_form(self):
         form = PaymentForm(user=self.user, data=data, request=self.request)
         self.assertTrue('organization' in form.fields)
 
+    def test_pycountry_validation_skipped_on_us_military_address(self):
```
Comment (waheedahmed) [inline ecommerce/extensions/payment/tests/test_forms.py]: nit: `test_form_validation_with_military_address`

**Review by waheedahmed — APPROVED** [review]
One minor comment otherwise 👍

**Comment by abutterworth** [comment]
Do we need a similar change in the frontend-app too? 
https://github.com/edx/frontend-app-payment/blob/master/src/payment/checkout/payment-form/utils/countryStatesMap.js#L17

**Comment by adeelehsan** [comment]
@abutterworth yes you need these changes in the frontend as well. Are we using the same backend view for payment MFE ?

**Pull request closed without merging**